devimint/src/main.rs

```diff
@@ -87,7 +87,7 @@ pub async fn latency_tests(dev_fed: DevFed) -> Result<()> {
     let iterations = 30;
     let mut reissues = Vec::with_capacity(iterations);
     for _ in 0..iterations {
-        let notes = cmd!(fed, "spend", "50000").out_json().await?["notes"]
+        let notes = cmd!(fed, "spend", "1000000").out_json().await?["notes"]
```

Comment:
This still isn't a whole lot :laughing: (1k sat)

Reply:
It has so many zeros already... :D

Maybe we need `ksats(100).to_string()`